tools/ldb/src/main.cpp

```diff
@@ -53,7 +53,7 @@ int main(int argc, const char** argv) {
     debuggerVM = new LSLuaState();
     debuggerVM->open();
 
-    lmLog(ldbLogGroup,"o executing LDB");
+    lmLog(ldbLogGroup, "Running");
```

Comment:
Because

```
       ldb  Running
```

seems nicer / less redundant than

```
       ldb  o executing LDB
```

It fits the surrounding messages more as well.